packages/build-info/src/frameworks/analog.ts

```diff
@@ -0,0 +1,26 @@
+import { BaseFramework, Category, Framework } from './framework.js'
+
+export class Analog extends BaseFramework implements Framework {
+  readonly id = 'analog'
+  name = 'Analog'
+  configFiles = []
+  npmDependencies = ['@analogjs/platform']
+  category = Category.SSG
```

Comment:
🙃

Reply:
The category classification might be a bit odd, but it matches with category we have for Next.js: https://github.com/netlify/build/blob/eed20eb2676d59d26ec2fb40335bb83c8a4194ed/packages/build-info/src/frameworks/next.ts#L8

Available categories are
https://github.com/netlify/build/blob/33dbdb40b7319f35b170e7389922466958ac578b/packages/build-info/src/frameworks/framework.ts#L8-L12

and SSG name is probably misleading as in essence it has both SSGs and overall meta frameworks - so kind of like ~fully featured thing allowing to build a non-SPA sites